oak_functions/examples/weather_lookup/module/src/tests.rs

```diff
@@ -108,9 +108,10 @@ async fn test_server() {
         .await
     });
 
+    // Test request coordinates are defined in `oak_functions/lookup_data_generator/src/data.rs`.
     {
         // Exact key_0.
-        let response = make_request(server_port, br#"{"lat":52.0,"lng":-0.01}"#)
+        let response = make_request(server_port, br#"{"lat":0.0,"lng":0.0}"#)
```

Comment:
I don't think the values in this test should change. It uses fixed custom lookup data (generated in lines 61-75), not data from the lookup data generator.

Reply:
Done